metrics.proto

```diff
@@ -30,52 +30,52 @@ enum MetricType {
 }
 
 message Gauge {
-  optional double value = 1;
+  double value = 1;
 }
 
 message Counter {
-  optional double value = 1;
+  double value = 1;
 }
 
 message Quantile {
-  optional double quantile = 1;
-  optional double value    = 2;
+  double quantile = 1;
+  double value    = 2;
 }
 
 message Summary {
-  optional uint64   sample_count = 1;
-  optional double   sample_sum   = 2;
+  uint64 sample_count = 1;
+  double sample_sum   = 2;
   repeated Quantile quantile     = 3;
 }
 
 message Untyped {
-  optional double value = 1;
+  double value = 1;
 }
 
 message Histogram {
-  optional uint64 sample_count = 1;
-  optional double sample_sum   = 2;
-  repeated Bucket bucket       = 3; // Ordered in increasing order of upper_bound, +Inf bucket is optional.
+  uint64 sample_count  = 1;
+  double sample_sum    = 2;
```

Comment:
Now either outdent the index number by one in these two or indent by one (to match below). It looks like we are indenting the indices enough to align them all in general.